tests/test_options.py

```diff
@@ -102,8 +104,23 @@ def test_transparency():
 
 
 @pytest.mark.sphinx("html", testroot="image")
-def test_image(directives: List[Tag]):
+def test_image(content: Sphinx, directives: List[Tag]):
     (img,) = directives
+    outdir = str(content.builder.outdir)
```

Comment:
Why is this being converted to a string?
Isn't it already?

Reply:
I thought this was a Path() - I will check this again